src/content/1.7/testing/_index.md

```diff
@@ -10,6 +10,70 @@ chapter: true
 # How testing works in PrestaShop
 {{< minver v="1.7.3" title="true" >}}
 
-Automated tests are located into `tests` folder. Tests are splitted into unit, integration and functional tests in PHP using PHPUnit testing framework. For end to end tests, we use Mocha.js and webdriver.io as bridge to control Selenium server.
+PrestaShop is a complex software and uses automated testing to ensure that the new additions to the codebase do not break existing behaviors.
 
+Automated tests are located into `tests` and `tests-legacy` folders.
 
+#### What kind of tests do PrestaShop use ?
```

Comment:
```suggestion
#### What kind of tests do PrestaShop use?
```

Reply:
Still not resolved :thinking:

Reply:
Done in https://github.com/PrestaShop/docs/pull/333/commits/788d058ed20cba6155b1e339b33707a4101e8aef